src/features/contactmoment/ContactmomentSwitcher.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <prompt-modal
+    :dialog="dialog"
+    message="Let op, je hebt het contactverzoek niet afgerond. Als je van contactmoment wisselt, wordt het contactverzoek niet verstuurd."
+  />
+  <details ref="detailsEl" v-if="contactmomentStore.contactmomenten.length > 1">
+    <summary class="utrecht-button">Contactmomenten</summary>
```

Comment:
This label might be a bit more descriptive like "Lopende contactmomenten", "Actieve contactmomenten", or something along those lines.